### PR TITLE
Add Vimb 3.1.0

### PR DESCRIPTION
**Pull request by aSourceFish**
I've taken this port from ports's history and modified it. But to reborn this port someone will need to handle CVS conflicts with the old www/vimb port.

### CODE REVIEW

File: www/vimb/pkg/PLIST [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@comment $OpenBSD: PLIST,v 1.2 2015/06/25 22:57:36 bcallah Exp $
+@bin bin/vimb
+lib/vimb/
+lib/vimb/webext_main.so
+share/applications/vimb.desktop
+share/man/
+share/man/man1/
+@man share/man/man1/vimb.1
```
Comment (ibara) [inline www/vimb/pkg/PLIST]: Wrong place. MANDIR is man, not share/man.

Comment (ibara) [reply www/vimb/pkg/PLIST]: Also missing the @exec and @unexec-delete lines for update-desktop-database (see any port with an RDEP on devel/desktop-file-utils).

File: www/vimb/Makefile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+HOMEPAGE =	http://fanglingsu.github.io/vimb/
+
+# GPLv3+
+PERMIT_PACKAGE_CDROM =	Yes
+
+WANTLIB += atk-1.0 c cairo gdk_pixbuf-2.0 gio-2.0 glib-2.0 gobject-2.0
+WANTLIB += intl pango-1.0 pangocairo-1.0 soup-2.4
+WANTLIB += cairo-gobject gdk-3 gtk-3 javascriptcoregtk-4.0 webkit2gtk-4.0
+
+LIB_DEPENDS =	www/webkitgtk4
```
Comment (ibara) [inline www/vimb/Makefile]: Missing
RUN_DEPENDS = devel/desktop-file-utils

File: www/vimb/pkg/PLIST (hunk 1 above)
Comment (aSourceFish) [reply www/vimb/pkg/PLIST]: ok, but I'll create new PR because I've removed my local repo

**Pull request closed without merging**